scripts/jobs/cron_tasks.inc.dns.20.pdns.php

```diff
@@ -104,23 +104,28 @@ private function _clearZoneTables($domains = null)
 
 		$del_rec_stmt = $this->pdns_db->prepare("DELETE FROM `records` WHERE `domain_id` = :did");
 		$del_meta_stmt = $this->pdns_db->prepare("DELETE FROM `domainmetadata` WHERE `domain_id` = :did");
-		$del_dom_stmt = $this->pdns_db->prepare("DELETE FROM `domains` WHERE `id` = :did");
+		$del_dom_stmt = $this->pdns_db->prepare("DELETE FROM `domains` WHERE `id` = :did"); 
 
 		foreach ($domains as $domain)
 		{
 			$pdns_domains_stmt->execute(array('domain' => $domain['domain']));
 			$pdns_domain = $pdns_domains_stmt->fetch(\PDO::FETCH_ASSOC);
-
-			$del_rec_stmt->execute(array('did' => $pdns_domain['id']));
-			$del_meta_stmt->execute(array('did' => $pdns_domain['id']));
-			$del_dom_stmt->execute(array('did' => $pdns_domain['id']));
+			// Only delete domains which have froxlor or nothing set as account
+			// so we don't delete domains not managed by us
+			if ($pdns_domain['account'] == 'froxlor' || $pdns_domain['account'] == '') {
+				$del_rec_stmt->execute(array('did' => $pdns_domain['id']));
+				$del_meta_stmt->execute(array('did' => $pdns_domain['id']));
+				$del_dom_stmt->execute(array('did' => $pdns_domain['id']));
+			} else {
+				$this->_logger->logAction(CRON_ACTION, LOG_INFO, 'Ignored domain '.$domain['domain'].' because we do not manage it');
+			}
 		}
 	}
 
 	private function _insertZone($domainname, $serial = 0)
 	{
 		$ins_stmt = $this->pdns_db->prepare("
-			INSERT INTO domains set `name` = :domainname, `notified_serial` = :serial, `type` = 'NATIVE'
+			INSERT INTO domains set `name` = :domainname, `type` = 'MASTER', `account` = 'froxlor'
```

Comment:
Right, should've probably added that to the message.
I removed it since that field is used by PDNS to remember which serial the slaves were last notified with. By manually changing it, PDNS doesn't know that there is a difference between the local serial and the serial of the slave server, so it wouldn't send out notifications to the slaves.